third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -154,14 +174,24 @@ func resourceComputeNetworkPeeringDelete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
-func findPeeringFromNetwork(network *compute.Network, peeringName string) *compute.NetworkPeering {
+func findPeeringFromNetwork(network *computeBeta.Network, peeringName string) *computeBeta.NetworkPeering {
 	for _, p := range network.Peerings {
 		if p.Name == peeringName {
 			return p
 		}
 	}
 	return nil
 }
+func expandNetworkPeering(d *schema.ResourceData) *computeBeta.NetworkPeering {
+	return &computeBeta.NetworkPeering{
+		ExportCustomRoutes:     d.Get("export_custom_routes").(bool),
```

Comment:
I think these should be within a `<% unless version = 'ga' -%>` block.

Reply:
Yes, it already was. I forgot to include this change in the last commit. Pushed it now